argo-cd-apps/app-of-apps/all-applications-staging.yaml

```diff
@@ -11,7 +11,8 @@ spec:
 
   source:
     path: argo-cd-apps/overlays/staging
-    repoURL: https://github.com/redhat-appstudio/infra-deployments.git
+    repoURL: https://github.com/sparkoo/infra-deployments.git
```

Comment:
Looks like you forgot to [revert](https://github.com/redhat-appstudio/infra-deployments#development-mode-for-your-own-clusters) development changes

Reply:
I'm keeping it here so reviewers can test it. I'll remove them before the merge. Or should I do it now?

Reply:
fixed